counterpartylib/lib/config.py

```diff
@@ -88,7 +88,7 @@
 
 
 DEFAULT_REQUESTS_TIMEOUT = 20  # 20 seconds
-
+DEFAULT_RPC_BATCH_SIZE = 300
```

Comment:
Shouldn't we then have the default value be bigger than the max number of transactions in a block? (So most users only see one query per block.) Except that we tell everyone to set `rpcthreads` to 1000. Should we increase that number? (It seems right now we should just set this to 1000. Then all *current* blocks will be done in one query: https://blockchain.info/charts/n-transactions-per-block)